### PR TITLE
Prisma SASE - Quarantine Host With Active Threat 

### PR DESCRIPTION
**Pull request by melamedbn**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
A new playbook for Prisma SASE and SentinelOne host quarantine.

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/PrismaAccess/Playbooks/playbook-Prisma_SASE_-_Quarantine_Host_With_Active_Threat.yml [hunk 1]
```diff
@@ -0,0 +1,429 @@
...
+
+  ## Playbook Flow
+  - **Trigger**: The playbook activates upon the creation of a new SentinelOne Threat incident.
+  - **Process**:
+    1. Retrieves the Host ID from Cortex Data Lake.
+    2. Loads detailed information about the agent.
+    3. If the Host ID is found and the agent is not decommissioned, it initiates quarantine through Prisma SASE.
+
+  ## Dependencies and Configuration
+  - This playbook requires being set as the default in the SentinelOne instance configuration.
```
Comment (altmannyarden) [inline Packs/PrismaAccess/Playbooks/playbook-Prisma_SASE_-_Quarantine_Host_With_Active_Threat.yml:16]: One of the dependencies is the CDL

File: Packs/PrismaAccess/Playbooks/playbook-Prisma_SASE_-_Quarantine_Host_With_Active_Threat.yml [hunk 2]
```diff
@@ -0,0 +1,429 @@
...
+  - It specifically responds to events categorized as 'Threats'.
+
+  **Note**: Ensure that the Prisma SASE, Cortex Data Lake and SentinelOne integrations are properly configured and operational for seamless execution of this playbook.
```
Comment (melamedbn) [inline Packs/PrismaAccess/Playbooks/playbook-Prisma_SASE_-_Quarantine_Host_With_Active_Threat.yml:19]: As appears here @altmannyarden

File: Packs/PrismaAccess/Playbooks/playbook-Prisma_SASE_-_Quarantine_Host_With_Active_Threat.yml [hunk 3]
```diff
@@ -0,0 +1,428 @@
...
+    isautoswitchedtoquietmode: false
+  "8":
+    id: "8"
+    taskid: ce581bda-eae3-4ae7-80b2-6172c932b51e
+    type: regular
+    task:
+      id: ce581bda-eae3-4ae7-80b2-6172c932b51e
+      version: -1
+      name: Load SentinelOne agent details
+      description: Loads a json from string input, and returns a json object result
```
Comment (ShirleyDenkberg) [inline Packs/PrismaAccess/Playbooks/playbook-Prisma_SASE_-_Quarantine_Host_With_Active_Threat.yml:263]:
```suggestion
      description: Loads a JSON from a string input, and returns a JSON object result.
```

File: Packs/PrismaAccess/Playbooks/playbook-Prisma_SASE_-_Quarantine_Host_With_Active_Threat.yml [hunk 4]
```diff
@@ -0,0 +1,428 @@
...
+    isautoswitchedtoquietmode: false
+  "9":
+    id: "9"
+    taskid: d729bc95-fdeb-4819-8349-8aef1a7c3ba5
+    type: regular
+    task:
+      id: d729bc95-fdeb-4819-8349-8aef1a7c3ba5
+      version: -1
+      name: 'Error: Could not quarantine host'
+      description: Prints an error entry with a given message
```
Comment (ShirleyDenkberg) [inline Packs/PrismaAccess/Playbooks/playbook-Prisma_SASE_-_Quarantine_Host_With_Active_Threat.yml:304]:
```suggestion
      description: Prints an error entry with a given message.
```

File: Packs/PrismaAccess/Playbooks/playbook-Prisma_SASE_-_Quarantine_Host_With_Active_Threat_README.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+
+This playbook does not use any sub-playbooks.
+
+### Integrations
+
+* PrismaSASE
```
Comment (ShirleyDenkberg) [inline Packs/PrismaAccess/Playbooks/playbook-Prisma_SASE_-_Quarantine_Host_With_Active_Threat_README.md:28]:
```suggestion
PrismaSASE
```

File: Packs/PrismaAccess/ReleaseNotes/2_1_3.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
+
+#### Playbooks
+
+##### New: Prisma SASE - Quarantine Host With Active Threat
+
+- New:
```
Comment (ShirleyDenkberg) [inline Packs/PrismaAccess/ReleaseNotes/2_1_3.md:6]:
```suggestion
New:
```

File: Packs/PrismaAccess/ReleaseNotes/2_1_3.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+  1. Retrieves the Host ID from Cortex Data Lake.
+  2. Loads detailed information about the agent.
+  3. If the Host ID is found and the agent is not decommissioned, it initiates quarantine through Prisma SASE.
+
+## Dependencies and Configuration
+- This playbook requires being set as the default in the SentinelOne instance configuration.
+- It specifically responds to events categorized as 'Threats'.
+- Ensure that the Prisma SASE, Cortex Data Lake and SentinelOne integrations are properly configured and operational for seamless execution of this playbook.
+ 
+(Available from Cortex XSOAR 6.10.0).
```
Comment (ShirleyDenkberg) [inline Packs/PrismaAccess/ReleaseNotes/2_1_3.md:21]: Is this information needed in the Release Notes?

**Comment by ShirleyDenkberg** [comment]
@melamedbn Doc review completed.

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**